debugger/src/test/resources/META-INF/plugin.xml

```diff
@@ -0,0 +1,10 @@
+<idea-plugin package="org.jetbrains.fortran" xmlns:xi="http://www.w3.org/2001/XInclude">
+    <id>org.rust.lang</id>
+    <xi:include href="/META-INF/fortran-core.xml" xpointer="xpointer(/idea-plugin/*)"/>
+
+    <!--suppress PluginXmlValidity -->
+    <content>
+        <module name="org.jetbrains.fortran.debugger"/>
```

Comment:
The module tag looks like an IDE-level tag, not a plugin-level https://intellij-support.jetbrains.com/hc/en-us/community/posts/360000563584-What-is-the-module-element-in-the-plugin-xml-for-. 
I am not sure that we want to specify them in our  code, I will try to rewrite XML config so that we do not  use it, we shouldn't copy Rust style here

Reply:
Hmm, reading the documentation there, that does seem what we want here, isn't it? The thing is that for `IU` the `nativeDebugger` has to be pulled in, while in `CL` the dependency on `com.intellij.modules.cidr.debugger`

```
A module represents a built-in plugin that is a non-removable part of a product. Some modules are available in all products, and some modules are available only in some, or even just one product. This section identifies and discusses modules of both types.
```

Maybe we should make it dependent on the IDE being built for, but in that case, the build step should be run for all combinations of `IDE` and `platfromVersion`.

Reply:
I applied changes regarding modules. We can depend on modules from IDE, but we shouldn't define them inside plugin and make one part of a plugin depend on another part of it